packages/extractor-vue/src/vue-extractor.ts

```diff
@@ -12,12 +12,17 @@ export const vueExtractor: ExtractorType = {
     onMessageExtracted,
     ctx: ExtractorCtx
   ) {
-    const { descriptor } = parse(code, {
+    const { descriptor, errors: parsedErrors } = parse(code, {
       sourceMap: true,
       filename,
       ignoreEmpty: true,
     })
 
+    if (parsedErrors.length) {
+      console.error("Error while parsing:")
+      throw parsedErrors
```

Comment:
Not sure if throwing error is the right approach. Should I simply log these errors and proceed with extraction?